addon/components/ember-flatpickr.js

```diff
@@ -10,11 +10,25 @@ import { getOwner } from '@ember/application';
  * Ember component that wraps the lightweight [`flatpickr`](https://flatpickr.js.org) datetime
  * chooser.
  *
- * This component extends Ember's `@ember/component/text-field` and mixes
- * in `ember-flatpickr/mixins/flatpickr`.
+ * The EmberFlatpickr component requires that you pass in at the very least a `date` property
+ * and a `onChange` callback.
  *
  * ```handlebars
- *  <EmberFlatpickr @date={{readonly model.someDate}}/>
+ *  <EmberFlatpickr @date={{model.someDate}} @onChange={{this.onChange}} />
+ * ```
+ *
+ * The EmberFlatpickr component internally makes use of ...attributes. This means that any
+ * native input attributes that you pass in should be passed in without the `@` symbol.
+ * For example
+ *
+ * ```handlebars
+ *  <EmberFlatpickr
+ *    @date={{model.someDate}}
+ *    @onChange={{this.onChange}}
+ *    placeholder="Pick a date"
+ *    aria-activedescendent="aria-activedescendent"
+ *    aria-autocomplete="aria-autocomplete"
+ *    aria-describedby="described by"/>
```

Comment:
HTML attributes should go first, but also is `aria-activedescendent="aria-activedescendent"` etc what we want to replace the bindings from before?

```suggestion
 *    aria-activedescendent="aria-activedescendent"
 *    aria-autocomplete="aria-autocomplete"
 *    aria-describedby="described by"
 *    placeholder="Pick a date"
 *    @date={{model.someDate}}
 *    @onChange={{this.onChange}}
 *    />
```

Reply:
For reference, these were the bindings being used in the old implementation

```
attributeBindings: [
    'aria-activedescendent',
    'aria-autocomplete',
    'aria-describedby',
    'aria-labelledby',
    'aria-multiline',
    'aria-placeholder',
    'aria-readonly',
    'aria-required'
  ],
```

I was looking through the upgrade guide for [glimmer](https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/) and the way to do it in the octane world is to have as a property in the component. Instead of storing those as properties in the component I thought it would best to just use `...attributes` to pass them in. I don't know if that answers your question.

Reply:
I'll resolve this conversation for now and you can reopen if need it be.